src/document-creator/document-creator.mdx

```diff
@@ -97,9 +118,7 @@ JSON.stringify(w)
 
 // result: "{\"address\":\"6a36c563a5350d7be66c801f901a67...\", ...}",
 ```
-
-Alternatively, you can refer to [Open Attestation's documentation](https://github.com/Open-Attestation/open-attestation-cli#wallet) to create your wallet.
-_Note: you will need to have Open Attestation Cli installed_
+This result can now be entered into the `"wallet"` field in the config file.
```

Comment:
config file maybe can link back to the example `(/document-creator#configuration-file)`